linux-stable.sh

```diff
@@ -245,12 +249,14 @@ git add . && git cherry-pick --continue"
             fi ;;
 
         "merge")
-            if ! GIT_MERGE_VERBOSITY=1 git merge --no-edit "v${TARGET_VERSION}"; then
+            if ! GIT_MERGE_VERBOSITY=1 git merge --edit -m "Merge ${TARGET_VERSION} into ${branch}" -m "Changes in ${TARGET_VERSION}: ($(git status | grep commits | cut -d ' ' -f8-9 | head -n1 | sed 's/commits./commits/'))" -m "$(tag_logs)" "v${TARGET_VERSION}" --no-edit; then
```

Comment:
Kill the boilerplate (this exact same message is used in both places) and make a function like in my own script ([`create_merge_message`](https://github.com/nathanchance/scripts/blob/master/merge-stable#L219-L230), [`append_conflicts`](https://github.com/nathanchance/scripts/blob/master/merge-stable#L233-L238)).

The git aliases can be viewed [here](https://github.com/nathanchance/scripts/blob/master/os/common#L16-L77) for translation.

Reply:
didn't check your personal scripts before, i just saw your commits on als repo xD, almost exact same
thanks

Reply:
Yeah didn't expect you to :P if you feel like adapting it for here, by all means feel free!